src/libs/actions/BankAccounts.js

```diff
@@ -193,10 +203,63 @@ function validateBankAccount(bankAccountID, validateCode) {
     });
 }
 
+/**
+ * @param {Object} data
+ * @returns {Object}
+ */
+function mergeWithLocalACHData(data) {
+    const updatedACHData = {
+        ...reimbursementAccountInSetup,
+        ...data,
+
+        // This param tells Web-Secure that this bank account is from NewDot so we can modify links back to the correct
+        // app in any communications. It also will be used to provision a customer for the Expensify card automatically
+        // once their bank account is successfully validated.
+        enableCardAfterVerified: true,
```

Comment:
I don't think we'll need this param anymore. Could you verify?

Reply:
We still use this param to automatically provision Ecards. We could move this to the API layer though, since every VBBA setup in NewDot passes this param.

Reply:
yes let's move it to php

Reply:
Web-E PR [here](https://github.com/Expensify/Web-Expensify/pull/34967)